extensions/amp-skimlinks/validator-amp-skimlinks.protoascii

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-skimlinks
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-skimlinks"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+
+tags: {  # <amp-skimlinks>
+  html_format: AMP
+  tag_name: "AMP-SKIMLINKS"
+  requires_extension: "amp-skimlinks"
+  attrs: {
+    name: "publisher-code"
+    mandatory: true
+  }
+  attrs: {
+    name: "link-selector"
+  }
+  attrs: {
+    name: "custom-tracking-id"
```

Comment:
it looks like there is set of restrictions on this attribute. would it be possible to codify them in a regex?

```
`custom-tracking-id` should be <=50 characters and only contain upper and lowercase characters, numbers, underscores and pipes.
```